RELEASES.md

```diff
@@ -0,0 +1,112 @@
+# Version 0.16.0 (2025-08-04)
+
+This release introduces initial support for the **LoongArch CPU architecture**, a major milestone for the project. Version 0.16.0 also significantly expands our Linux ABI compatibility with the addition of **nine new system calls** such as `memfd_create` and `pidfd_open`.
+
+Key enhancements include expanded functionality for **netlink (for uevent notifications)** and **UNIX sockets (for file descriptor passing)**, the first implementation of **CgroupFS**, and a major testing improvement with the integration of system call tests from the **Linux Test Project (LTP)**. We've also adopted **[Nix](https://nix.dev/manual/nix/2.28/introduction)** for building the initramfs, streamlining our cross-compilation and testing workflow.
+
+## Asterinas Kernel
+
+We have made the following key changes to the Asterinas kernel:
+
+* New system calls or features:
+    * Memory:
+        * [Add the `mremap` system call](https://github.com/asterinas/asterinas/pull/2162)
+        * [Add the `msync` system call based on an inefficient implementation](https://github.com/asterinas/asterinas/pull/2154)
+        * [Add the `memfd_create` system call](https://github.com/asterinas/asterinas/pull/2149)
+    * Processes and IPC:
+        * [Add the `pidfd_open` system call along with the `CLONE_PIDFD` flag](https://github.com/asterinas/asterinas/pull/2151)
+    * File systems and I/O in general:
+        * [Add the `close_range` system call](https://github.com/asterinas/asterinas/pull/2128)
+        * [Add the `fadvise64` system call (dummy implementation)](https://github.com/asterinas/asterinas/pull/2125)
+        * [Add the `ioprio_get` and `ioprio_set` system calls (dummy implementation)](https://github.com/asterinas/asterinas/pull/2126)
+    * Sockets and network:
+        * [Add the `epoll_pwait2` system call](https://github.com/asterinas/asterinas/pull/2123)
+* Enhanced system calls or features:
+    * Processes:
+        * [Add `FUTEX_WAKE_OP` support for the `futex` system call](https://github.com/asterinas/asterinas/pull/2146)
+        * [Add `WSTOPPED` and `WCONTINUED` support to the `wait4` and `waitpid` system calls](https://github.com/asterinas/asterinas/pull/2166)
+        * [Add more fields in `/proc/*/stat` and `/proc/*/status`](https://github.com/asterinas/asterinas/pull/2215)
```

Comment:
Why is ‘Add more fields in /proc/*/stat and /proc/*/status’ not included under ‘File systems and I/O in general’? These should be considered entries under procfs.

Reply:
Same reason as I put cgroupfs under the security category. Technically, the modification is done to a FS (procfs), but its purpose to expose more process-related information. I think purpose is more important than means. On the other hand, if we have a procfs category, then I would definitely put it under that category.